src/index.js

```diff
@@ -397,7 +397,7 @@ class Jimp extends EventEmitter {
             return throwError.call(this, 'cb must be a function', cb);
         }
 
-        const mime = MIME.getType(path);
+        const mime = MIME.getType(path) || this.getMime();
```

Comment:
This isn't the name of the function. It's actually `getMIME`

Reply:
you also need to `MIME.getExtension(mime)` and add it to the path. But also if it has an extension you can't just add it.

```
        let mime = MIME.getType(path);

        if (!mime && !Path.extname(path)) {
          mime = this.getMIME();
          path += `.${MIME.getExtension(mime)}`
        }
```